functions/go/set-namespace/go.mod

```diff
@@ -2,7 +2,11 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-na
 
 go 1.17
 
-require github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220506190241-f85503febd54
+require (
+	github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220706221933-7181f451a663
+	k8s.io/api v0.24.2
+	k8s.io/apimachinery v0.24.2
```

Comment:
We should try to use the same version as porch. Otherwise, it can cause problems when we try to import it in Porch.

Reply:
This is really a good point! Do you have any ideas how we can guarantee it automatically? This requirement makes sense to me, but it is hard to be sustainable.

Reply:
I manually updated this for now.